survey.html

```diff
@@ -105,13 +105,13 @@ <h1>TolakedCFC Survey Form</h1>
                 <label for="React">
```

Comment:
You dont have to embed your input in your label tag, you can use a div to wrap them both if necessary